src/gfx_device/gfx_device.rs

```diff
@@ -59,7 +59,7 @@ impl GfxDevice {
         // the scene.
         for (rend, entity) in (&renderables, &entities).iter() {
             let global_trans = match global_transforms.get(entity) {
-                Some(gt) => gt.clone(),
+                Some(gt) => *gt,
```

Comment:
I don't think it's good idea
![screenshot from 2017-03-05 21 32 48](https://cloud.githubusercontent.com/assets/6566674/23590129/6fe9cd9e-01eb-11e7-99c3-2b293e24171f.png)